docs/guides/cloud/introduction.mdx

```diff
@@ -4,157 +4,152 @@ sidebar_position: 10
 sidebar_label: Introduction
 ---
 
-:::tip
+[Cypress Cloud](https://on.cypress.io/cloud) is our enterprise-ready companion
+to the open-source Cypress app, allowing you to:
+
+- [View past test results](#View-past-test-results) from your CI environment.
+- [Analyze and diagnose](#Analyze-and-diagnose-test-health) test failures.
+- [Reduce test time and save money](#Reduce-test-time--CI-costs) by smartly
+  orchestrating future runs across multiple machines.
+- [Integrate with your favorite developer tools](#Integrate-with-your-favorite-tools)
+  to help streamline your workflows.
+- [View CI runs and test health](#Cypress-app-integration) directly from the
+  Cypress app.
+- [Configure Cypress Cloud how you want](#Flexible-enterprise-configuration-and-single-sign-on) and enable enterprise single sign on.
+
+[Get started with Cypress Cloud](/guides/cloud/getting-started) for free now, or
+read on for more benefits of Cypress Cloud.
 
-<strong>
-  <Icon name="star" /> Get Started
-</strong>
-
-To get started with Cypress Cloud,
-[sign up here](https://cloud.cypress.io/signup) for our
-[free plan](https://www.cypress.io/pricing) which comes with 3 users and 500
-monthly test results.
-
-:::
-
-[Cypress Cloud](https://on.cypress.io/cloud) is our enterprise-ready, web-based
-companion to the Cypress app. It gives you online access to your recorded test
-results, orchestrates test runs across multiple machines, provides rich
-analytics and diagnostics, and integrates those insights with your favorite
-tools.
-
-<!-- textlint-disable -->
-
-<DocsVideo src="https://vimeo.com/736212167" />
-
-<!-- textlint-enable -->
-
-:::info
-
-##### <Icon name="graduation-cap" /> Real World Example <Badge type="success">New</Badge>
-
-The Cypress
-[Real World App (RWA)](https://github.com/cypress-io/cypress-realworld-app)
-leverages [Cypress Cloud in CI](https://cloud.cypress.io/projects/7s5okt) to
-test over 300 test cases in parallel across 25 machines, multiple browsers,
-multiple device sizes, and multiple operating systems.
+## Benefits
 
-Check out the <Icon name="github" />
-[Real World App in Cypress Cloud](https://cloud.cypress.io/projects/7s5okt).
+### View past test results
 
-:::
+Each test run is stored in Cypress Cloud, where you can see past results and the
+current state of your app on the [Latest Runs](/guides/cloud/recorded-runs) page. View
+each test's
+[command logs, screenshots, video replays, stack traces, and CI logs](/guides/cloud/recorded-runs#Test-detail-sidebar).
+Quickly identifying a test failure in CI is just a click away.
 
-## Benefits
+<DocsVideo src="https://vimeo.com/806005654" />
```

Comment:
We have video hosted on YT & Vimeo. I was never sure which was the preference. Are these on Vimeo for any particular reason or just whatever was quick?

Reply:
A couple I can think of:

- For these videos, they would make no sense at all if someone randomly came to them on YT. 
- We wouldn't want the YT UI interface to pop up and take them to other suggested videos and whatnot on these videos